src/control/server/mgmt_system_test.go

```diff
@@ -213,6 +213,7 @@ func TestServer_MgmtSvc_LeaderQuery(t *testing.T) {
 				if leader, _, _ := db.LeaderQuery(); leader != "" {
 					break
 				}
+				time.Sleep(250 * time.Millisecond)
```

Comment:
why the sleep?

Reply:
Without the sleep, `db.LeaderQuery()` is called in a tight loop. That was intentional because I wanted the test to finish as quickly as it could. As I was profiling the system tests to determine the source of the long run times I noticed that this particular call was using a lot of CPU time, so I decided that waiting for a quarter of a second between iterations was fine.